common-lib/src/cmr/common/generics_documentation.clj

```diff
@@ -0,0 +1,126 @@
+
+(ns cmr.common.generics-documentation
+  "Defines utilities for new generic document pipeline. Most functions will deal
+   with either returning generic config files, or lists of approved generics."
+  (:require
+   [cheshire.core :as json]
+   [clojure.java.io :as io]
+   [clojure.set :as set]
+   [clojure.string :as string]
+   [cmr.common.config :as cfg]
+   [cmr.common.generics :as gconfig]
+   [cmr.common.log :as log :refer [error, info]]
+   [inflections.core :as inf]))
+
+(defn latest-approved-documentation
```

Comment:
If latest-approved-documentation isn't needed then we don't need this function either.